specification/compute/resource-manager/Microsoft.Compute/stable/2021-04-01/compute.json

```diff
@@ -8321,6 +8970,253 @@
       ],
       "description": "The list dedicated host operation response."
     },
+    "CapacityReservationGroupInstanceView": {
+      "type": "object",
+      "properties": {
+        "capacityReservations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CapacityReservationInstanceViewWithName"
+          },
+          "readOnly": true,
+          "description": "List of instance view of the capacity reservations under the capacity reservation group."
+        }
+      }
+    },
+    "CapacityReservationGroupProperties": {
+      "type": "object",
+      "properties": {
+        "capacityReservations": {
```

Comment:
missed Capitalization

Reply:
This is a property name. So followed camel casing like all other properties.

Reply:
i see. ty